package.json

```diff
@@ -0,0 +1,28 @@
+{
```

Comment:
Try getting in the habit of filling out your script section during you "setup" stage.